upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -538,6 +539,23 @@ func (tf *TemplateFunctions) OpenStackCCMTag() string {
 	return tag
 }
 
+// AWSCCMTag returns the correct tag for the cloud controller manager based on
+// the Kubernetes Version
+func (tf *TemplateFunctions) AWSCCMTag() string {
+	var tag string
+	parsed, err := util.ParseKubernetesVersion(tf.Cluster.Spec.KubernetesVersion)
+
+	// Update when we have stable releases
+	if err != nil {
+		tag = "v1.19.0-alpha.1"
```

Comment:
Should we log an info message here to inform that parsing failed and we're defaulting to this tag?

Reply:
Yeah I'll add that.